dao/src/main/java/org/n52/series/db/beans/DataEntity.java

```diff
@@ -234,13 +234,16 @@ public void setChild(boolean child) {
         this.child = child;
     }
 
-    public Set<Parameter< ? >> getParameters() {
+    public Set<Parameter> getParameters() {
         return parameters;
     }
 
-    public void setParameters(Set<Parameter< ? >> parameters) {
-        if (parameters != null) {
-            this.parameters.addAll(parameters);
+    @SuppressWarnings(value = "unchecked")
+    public void setParameters(Object parameters) {
+        if (parameters instanceof Set<?>) {
```

Comment:
Hibernate had some problems with the former setter (that's why I merged Carsten's solution). However, after double checking it seems, that using `this.parameters.addAll(parameters);` within proxied POJO does not make sense ... AFAIR, I added this because of some findbugs rule about mutable collection